packages/admin/src/table/filterbar/FilterBarPopOverFormField.tsx

```diff
@@ -0,0 +1,232 @@
+import { useIntl } from "react-intl";
+
+import { Field } from "../../form/Field";
+import { FilterBarActiveFilterBadge } from "./FilterBarActiveFilterBadge";
+
+export interface PopoverFormFieldThemeProps {
+    submitButton?: ButtonProps;
+    resetButton?: ButtonProps;
+}
+
+interface StyleProps {
+    fieldBarWidth: number;
+}
+
+export type CometAdminFilterBarPopOverFormFieldClassKeys =
+    | "root"
+    | "styledBox"
+    | "labelWrapper"
+    | "popoverContentContainer"
+    | "popoverInnerContentContainer"
+    | "paper"
+    | "buttonsContainer"
+    | "submitContainer"
+    | "resetCloseContainer";
+
+const useStyles = makeStyles(
+    (theme: Theme) => ({
+        root: {
+            position: "relative",
+        },
+        fieldBarWrapper: {
+            minWidth: (props: StyleProps) => `${props.fieldBarWidth}px`,
+            border: `1px solid ${theme.palette.grey[300]}`,
+            position: "relative",
+            marginBottom: "10px",
+            marginRight: "10px",
+        },
+        styledBox: {
+            position: "relative",
+            alignItems: "center",
+            padding: "10px 20px",
+            cursor: "pointer",
+            display: "flex",
+
+            "&:after": {
+                borderTop: `4px solid ${theme.palette.grey[300]}`,
+                borderRight: "4px solid transparent",
+                borderLeft: "4px solid transparent",
+                position: "absolute",
+                display: "block",
+                right: "10px",
+                content: "''",
+                top: "50%",
+                height: 0,
+                width: 0,
+            },
+        },
+        labelWrapper: {
+            marginRight: "15px",
+            boxSizing: "border-box",
+        },
+        popoverContentContainer: {
+            "& [class*='CometAdminFormFieldContainer-root']": {
+                boxSizing: "border-box",
+                padding: "20px",
+                marginBottom: 0,
+            },
+        },
+        popoverInnerContentContainer: {
+            minWidth: 300,
+        },
+        buttonsContainer: {
+            borderTop: `1px solid ${theme.palette.grey[300]}`,
+            justifyContent: "space-between",
+            padding: "10px 15px",
+            display: "flex",
+        },
+        submitContainer: {},
+        resetContainer: {
+            marginRight: "15px",
+        },
+        paper: {
+            border: "1px solid grey",
+            marginLeft: -1, //due to border of popover, but now overrideable with styling if needed
+        },
+    }),
+    { name: "CometAdminFilterBarPopOverFormField" },
+);
+
+export interface FilterBarPopOverFormFieldProps {
+    label: string;
+    name: string;
+    dirtyFieldsBadge?: (fieldState?: FieldState<any>) => React.Component;
+    calcNumberDirtyFields?: (fieldState?: FieldState<any>) => number;
+    fieldBarWidth: number;
+}
+
+export function FilterBarPopOverFormField({
```

Comment:
In my opinion, the name `FilterBar.PopOverFormField` exposes a technicality (that MUI Popover ist used), and is difficult to remember. `FilterBar.Filter` or simply `FilterBar.Field` would be easier to remember, while keeping the same information (that it's a field in the filter bar)

Reply:
good point. I wouldn't use `Field`  - as a filter can contain multiple fields